wps-common/src/main/java/au/org/aodn/aws/wps/status/ExecuteStatusBuilder.java

```diff
@@ -101,6 +118,79 @@ public static void main(String[] args) {
         System.out.println();
         ExecuteStatusBuilder e = new ExecuteStatusBuilder();
     }
+
+
+    public static ProcessFailedType getProcessFailedType(String message, String code)
+    {
+        ProcessFailedType failed = new ProcessFailedType();
+        failed.setExceptionReport(StatusHelper.getExceptionReport(message, code));
+        return failed;
+    }
+
+
+    public static ProcessStartedType getProcessStartedType(String message, Integer percentComplete)
+    {
+        ProcessStartedType started = new ProcessStartedType();
+        started.setValue(message);
+        started.setPercentCompleted(percentComplete);
+        return started;
+    }
+
+
+    public static void addOutputToResponse(ExecuteResponse response, String outputIdentifier, String outputHref)
```

Comment:
Could these two methods be private? ExecuteResponse and other jaxb classes should be left as behind the scenes implementation details.

Reply:
Changed to private.